packages/manager/modules/web-paas/src/web-paas.routing.js

```diff
@@ -0,0 +1,45 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('web-paas', {
+    url: '/paas/webpaas/projects',
```

Comment:
```suggestion
    url: '/paas/webpaas',
```
List should be available at root state to be consistent